easybuild/easyconfigs/o/onedrive/onedrive-2.4.11-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'onedrive'
+version = '2.4.11'
+
+homepage = 'https://abraunegg.github.io/'
+description = """
+A free Microsoft OneDrive Client which supports OneDrive Personal, OneDrive for Business,
+OneDrive for Office365 and SharePoint."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/abraunegg/onedrive/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['839ec89c0b2a2c71db19668f7d333f9ef5d6910451b49fba1c2dc8e53885aa2f']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('LDC', '1.25.1'),
+]
+
+dependencies = [
+    ('cURL', '7.72.0'),
+    ('SQLite', '3.33.0'),
+]
+
+_compldir = '%(installdir)s/share/tab_completion'
```

Comment:
@smoors oughtn't this to have the `local` prefix?

Reply:
@verdurin both `_` and `local_` prefixes are allowed, see https://docs.easybuild.io/en/latest/Easyconfig-files-local-variables.html#recommended-naming-scheme-for-local-variables-in-easyconfig-files

Reply:
Thought you were unlikely to have got this wrong, but better to ask...

Reply:
yeah, the tests would probably fail it it was wrong.